notebooks/ch10/modeling_abs_huber.md

```diff
@@ -1,6 +1,6 @@
 
 <h1>Table of Contents<span class="tocSkip"></span></h1>
```

Comment:
Great job with this file! All of the thetas are correct, and you remembered to bold all of the vectors :)